Data/models/Model.xcdatamodeld/Model9.xcdatamodel/contents

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
This change needs to be created on 1.6 first, and the new model8 copied into here.